### PR TITLE
python-prompt_toolkit: update to 2.0.9.

### PR DESCRIPTION
**Pull request by AluisioASG**
Also move the current version to `python-prompt_toolkit1`, because some of its dependents don't support v2.

### CODE REVIEW

File: srcpkgs/python-prompt_toolkit1/template [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+version=1.0.15
+revision=1
+noarch=yes
+wrksrc="prompt_toolkit-${version}"
+build_style=python-module
+pycompile_module="prompt_toolkit"
+hostmakedepends="python-setuptools python3-setuptools"
+depends="python-six python-wcwidth"
+short_desc="Python2 library for building powerful interactive command lines"
+maintainer="Alessio Sergi <al3hex@gmail.com>"
```
Comment (maxice8) [inline srcpkgs/python-prompt_toolkit1/template]: use your own name

File: srcpkgs/python-prompt_toolkit1/template [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+license="3-clause-BSD"
```
Comment (maxice8) [inline srcpkgs/python-prompt_toolkit1/template]: BSD-3-Clause

File: srcpkgs/python-prompt_toolkit1/template [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+checksum=858588f1983ca497f1cf4ffde01d978a3ea02b01c8a26a8bbc5cd2e66d816917
+conflicts="python-prompt_toolkit>=2"
+replaces="python-prompt_toolkit<2"
+
+post_install() {
+	vlicense LICENSE
+}
+
+python3-prompt_toolkit1_package() {
+	noarch=yes
```
Comment (maxice8) [inline srcpkgs/python-prompt_toolkit1/template]: archs=noarch

File: srcpkgs/python-prompt_toolkit1/template [hunk 4]
```diff
@@ -0,0 +1,35 @@
+# Template file for 'python-prompt_toolkit1'
+pkgname=python-prompt_toolkit1
+version=1.0.15
+revision=1
+noarch=yes
```
Comment (maxice8) [inline srcpkgs/python-prompt_toolkit1/template]: archs=noarch

File: srcpkgs/python-prompt_toolkit/template [hunk 5]
```diff
@@ -1,31 +1,43 @@
...
 build_style=python-module
 pycompile_module="prompt_toolkit"
 hostmakedepends="python-setuptools python3-setuptools"
 depends="python-six python-wcwidth"
+checkdepends="python-pytest python-six python-wcwidth python3-pytest
+ python3-six python3-wcwidth"
 short_desc="Python2 library for building powerful interactive command lines"
 maintainer="Alessio Sergi <al3hex@gmail.com>"
-homepage="https://github.com/jonathanslenders/python-prompt-toolkit"
 license="3-clause-BSD"
```
Comment (maxice8) [inline srcpkgs/python-prompt_toolkit/template]: BSD-3-Clause

File: srcpkgs/python-jupyter_console/template [hunk 6]
```diff
@@ -1,13 +1,13 @@
 # Template file for 'python-jupyter_console'
 pkgname=python-jupyter_console
 version=5.2.0
-revision=2
+revision=3
 noarch=yes
```
Comment (maxice8) [inline srcpkgs/python-jupyter_console/template]: archs=noarch

File: srcpkgs/python-ipython/template [hunk 7]
```diff
@@ -1,14 +1,14 @@
 # Template file for 'python-ipython'
 pkgname=python-ipython
 version=5.8.0
-revision=1
+revision=2
 noarch=yes
```
Comment (maxice8) [inline srcpkgs/python-ipython/template]: archs=noarch

**Comment by AluisioASG** [comment]
@xaltsc got it further along already.

**Pull request closed without merging**